Src/Web/Web.Shared.Net.Tests/Helpers/HttpModuleHelper.cs

```diff
@@ -67,7 +67,9 @@ public static HttpContext GetFakeHttpContext(IDictionary<string, string> headers
 
             var workerRequest = new SimpleWorkerRequestWithHeaders(UrlPath, UrlQueryString, new StringWriter(CultureInfo.InvariantCulture), headers);
             
-            return new HttpContext(workerRequest);
+            var context = new HttpContext(workerRequest);
+            HttpContext.Current = context;
```

Comment:
Where is this cleaned up?

---
In reply to: [110214089](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/413#discussion_r110214089) [](ancestors = 110214089)